RELEASING.md

```diff
@@ -0,0 +1,41 @@
+## Releasing Shopify CLI
+
+1. Check the Semantic Versioning page for info on how to version the new release: http://semver.org
+2. Create a branch named `release_X_Y_Z` (replacing `X_Y_Z` with the intended release version)
+  ```
+  $ git checkout -b release_X_Y_Z
+  ```
+3. Update the version of Shopify CLI in `lib/shopify-cli/version.rb`
+4. Add an entry for the new release to `CHANGELOG.md`
+5. Commit the changes with a commit message like "Packaging for release X.Y.Z"
+  ```
+  $ git commit -am "Packaging for release X.Y.Z"
+  ```
+6. Push out the changes
+  ```
+  $ git push -u origin release_X_Y_Z
+  ```
+7. Open a PR for the branch, get necessary approvals from code owners and merge into main branch
+8. Deploy using Shipit
+9. On local machine and _AFTER_ gem has been published to https://rubygems.org, run
+  ```
+  $ rake package
+  ```
+  This will generate the `.deb`, `.rpm` and brew formula files, which will be located in `packaging/builds/X.Y.Z/`.
+10. Clone the `Shopify/homebrew-shopify` repository (if not already cloned), and then
+    * create a branch named `release_X_Y_Z_of_shopify-cli`
+    * update the brew formula in `shopify-cli.rb` with the generated formula in `packaging/builds/X.Y.Z/` in the `Shopify/shopify-app-cli` repo (from step 9)
+    * commit the change and create a PR
+    * when PR is approved, merge into main branch of `Shopify/homebrew-shopify` repository
+11. Go to releases page of `Shopify/shopify-app-cli` repo (https://github.com/Shopify/shopify-app-cli/releases) and create a new release:
```

Comment:
Minor: we could make the text slightly shorter here by making `releases page` into a link to the actual page so we don't need the entire URL in the paragraph.

Reply:
👍  Changed!